source/fem-dem/vans_assemblers.cc

```diff
@@ -1155,6 +1155,92 @@ GLSVansAssemblerKochHill<dim>::calculate_particle_fluid_interactions(
 template class GLSVansAssemblerKochHill<2>;
 template class GLSVansAssemblerKochHill<3>;
 
+template <int dim>
+void
+GLSVansAssemblerBeetstra<dim>::calculate_particle_fluid_interactions(
+  NavierStokesScratchData<dim> &scratch_data)
+
+{
+  unsigned int particle_number;
+  double       normalized_drag_force = 0;
+  auto &       beta_drag             = scratch_data.beta_drag;
+
+  Tensor<1, dim> superficial_velocity;
+  Tensor<1, dim> relative_velocity;
+  Tensor<1, dim> drag_force;
+
+
+  // Physical Properties
+  Assert(
+    !scratch_data.properties_manager.is_non_newtonian(),
+    RequiresConstantViscosity(
+      "GLSVansAssemblerBeetstra<dim>::calculate_particle_fluid_interactions"));
+  const double viscosity = scratch_data.properties_manager.viscosity_scale;
+
+  Assert(
+    scratch_data.properties_manager.density_is_constant(),
+    RequiresConstantDensity(
+      "GLSVansAssemblerBeetstra<dim>::calculate_particle_fluid_interactions"));
+  const double density = scratch_data.properties_manager.density_scale;
+
+  const auto pic  = scratch_data.pic;
+  beta_drag       = 0;
+  particle_number = 0;
+
+  // Loop over particles in cell
+  for (auto &particle : pic)
+    {
+      auto particle_properties = particle.get_properties();
+
+      superficial_velocity =
+        scratch_data.fluid_velocity_at_particle_location[particle_number];
+
+      relative_velocity =
+        scratch_data.fluid_velocity_at_particle_location[particle_number] -
+        scratch_data.particle_velocity[particle_number];
+
+      double cell_void_fraction =
```

Comment:
is this the cell void fraction or the void fraction interpolated at the particle location?

Reply:
It is still the interpolated at the particle location